darwin/importer/formats/coco.py

```diff
@@ -48,12 +48,18 @@ def parse_annotation(annotation: Dict[str, Any], category_lookup_table: Dict[str
         print("Warning, unsupported RLE, skipping")
         return None
 
+    try:
+        extra_text = annotation["extra"]["text"]
```

Comment:
Is this `extra` field part of the original COCO format? If so, could you please send over a piece of documentation that would prove that?